LanguageFeatures/nnbd/const_evaluation_A11_t02.dart

```diff
@@ -0,0 +1,103 @@
+/// @author sgrekhov22@gmail.com
+
+import "dart:async";
+import "../../Utils/expect.dart";
+
+main() {
+  const cNull = <Null>[];
+  const cNever = <Never>[];
+  const cDynamic = <dynamic>[];
+  const cObject = <Object>[];
+  const cNullableObject = <Object?>[];
+  const cVoid = <void>[];
+  const cFutureOrVoid = <FutureOr<void>>[];
+  const cFutureOrDynamic = <FutureOr<dynamic>>[];
+  const cFutureOrObject = <FutureOr<Object>>[];
+  const cFutureOrNullableObject = <FutureOr<Object?>>[];
+  const cFutureOrNull = <FutureOr<Null>>[];
+
+  Expect.notEquals(cNull, cNever);
+  Expect.notEquals(cNull, cDynamic);
+  Expect.notEquals(cNull, cObject);
+  Expect.notEquals(cNull, cNullableObject);
+  Expect.notEquals(cNull, cVoid);
+  Expect.notEquals(cNull, cFutureOrVoid);
+  Expect.notEquals(cNull, cFutureOrDynamic);
+  Expect.notEquals(cNull, cFutureOrObject);
+  Expect.notEquals(cNull, cFutureOrNullableObject);
+  Expect.notEquals(cNull, cFutureOrNull);
+
+  Expect.notEquals(cNever, cDynamic);
+  Expect.notEquals(cNever, cObject);
+  Expect.notEquals(cNever, cNullableObject);
+  Expect.notEquals(cNever, cVoid);
+  Expect.notEquals(cNever, cFutureOrVoid);
+  Expect.notEquals(cNever, cFutureOrDynamic);
+  Expect.notEquals(cNever, cFutureOrObject);
+  Expect.notEquals(cNever, cFutureOrNullableObject);
+  Expect.notEquals(cNever, cFutureOrNull);
+
+  Expect.notEquals(cDynamic, cObject);
+  Expect.notEquals(cDynamic, cNullableObject);
+  Expect.notEquals(cDynamic, cVoid);
+  Expect.notEquals(cDynamic, cFutureOrVoid);
+  Expect.notEquals(cDynamic, cFutureOrObject);
+  Expect.notEquals(cDynamic, cFutureOrNullableObject);
+  Expect.notEquals(cDynamic, cFutureOrNull);
+
+  Expect.notEquals(cObject, cNullableObject);
+  Expect.notEquals(cObject, cVoid);
+  Expect.notEquals(cObject, cFutureOrVoid);
+  Expect.notEquals(cObject, cFutureOrDynamic);
+  Expect.notEquals(cObject, cFutureOrNullableObject);
+  Expect.notEquals(cObject, cFutureOrNull);
+
+  Expect.notEquals(cNullableObject, cVoid);
+  Expect.notEquals(cNullableObject, cFutureOrVoid);
+  Expect.notEquals(cNullableObject, cFutureOrDynamic);
+  Expect.notEquals(cNullableObject, cFutureOrObject);
+  Expect.notEquals(cNullableObject, cFutureOrNull);
+
+  Expect.notEquals(cVoid, cFutureOrDynamic);
+  Expect.notEquals(cVoid, cFutureOrObject);
+  Expect.notEquals(cVoid, cFutureOrNullableObject);
+  Expect.notEquals(cVoid, cFutureOrNull);
+
+  Expect.notEquals(cFutureOrVoid, cFutureOrDynamic);
+  Expect.notEquals(cFutureOrVoid, cFutureOrObject);
+  Expect.notEquals(cFutureOrVoid, cFutureOrNullableObject);
+  Expect.notEquals(cFutureOrVoid, cFutureOrNull);
+
+  Expect.notEquals(cFutureOrDynamic, cFutureOrObject);
+  Expect.notEquals(cFutureOrDynamic, cFutureOrNullableObject);
+  Expect.notEquals(cFutureOrDynamic, cFutureOrNull);
+
+  Expect.notEquals(cFutureOrObject, cFutureOrNullableObject);
+  Expect.notEquals(cFutureOrObject, cFutureOrNull);
+
+  Expect.notEquals(cFutureOrNullableObject, cFutureOrNull);
```

Comment:
Again, there's no need to expand this one even though LanguageFeatures/nnbd/const_evaluation_A10_t01.dart may be expanded somewhat.